.github/dependabot.yml

```diff
@@ -0,0 +1,36 @@
+# #############################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Comment:
I think you need to update the year in your copyright header template 😁

Reply:
Done, thx!